server/oauth.go

```diff
@@ -1,63 +1,77 @@
 package server
 
 import (
-	"context"
 	"fmt"
 	"net/http"
-	"strconv"
+
+	"github.com/google/jsonapi"
+	"github.com/labstack/echo/v4"
 
 	"github.com/bytebase/bytebase/api"
 	"github.com/bytebase/bytebase/common"
 	vcsPlugin "github.com/bytebase/bytebase/plugin/vcs"
-	"github.com/google/jsonapi"
-	"github.com/labstack/echo/v4"
 )
 
 func (s *Server) registerOAuthRoutes(g *echo.Group) {
-	g.POST("/oauth/vcs/:vcsID/exchange-token", func(c echo.Context) error {
-		ctx := context.Background()
-
-		vcsID, err := strconv.Atoi(c.Param("vcsID"))
-		if err != nil {
-			return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Failed to marshal oauth provider's ID: %v", c.Param("id"))).SetInternal(err)
+	g.POST("/oauth/vcs/exchange-token", func(c echo.Context) error {
```

Comment:
nit: Since this endpoing have two usages, maybe it is more favorable to add a comment here to explain the different between them?

Reply:
Added comment in https://github.com/bytebase/bytebase/pull/1007/commits/900ce3e184e23971f76ee2162dd3b09e50ddadc0